src/dune/dune_file.ml

```diff
@@ -1053,14 +1035,32 @@ module Install_conf = struct
     { section : Install.Section.t
     ; files : File_binding.Unexpanded.t list
     ; package : Package.t
+    ; enabled_if : Blang.t
     }
 
   let decode =
     fields
       (let+ section = field "section" Install.Section.decode
        and+ files = field "files" File_binding.Unexpanded.L.decode
-       and+ package = Pkg.field "install" in
-       { section; files; package })
+       and+ package = Pkg.field "install"
+       and+ enabled_if =
+         let allowed_vars =
+           Some
+             (List.map
+                ~f:(function
+                  | var -> (var, (2, 6)))
```

Comment:
Why not just `fun var ->`?

Reply:
It was to stress the fact that other variables can be assigned different minimal version.
But I will factorize it with the list.